Library/ViewModels/DiscoveryPageViewModel.swift

```diff
@@ -424,6 +424,10 @@ private func saveSeen(activities: [Activity]) {
 }
 
 private func refTag(fromParams params: DiscoveryParams, project _: Project) -> RefTag {
+  if params.tagId != nil, params.tagId == .goRewardless {
```

Comment:
It seems `RefTag`s are inferred in this function. I realize that `Project` is not needed but I am still trying to change as little as possible 😅

Reply:
Any reason you've chosen to hard-code the `tagId` type here? Couldn't you just pass in the `tagId` to the `.projectCollection` refTag?

```
if let tagId = params.tagId {
   return .projectCollection(tagId)
}
```